pigeon/static/css/sweetalert.css

```diff
@@ -1 +0,0 @@
-../node_modules/sweetalert/dist/sweetalert.css
```

Comment:
Hum why deleting this one?

Reply:
The link was broken, yarn didn't fetch it (the sweetalert folder in node_modules only contains js). I assumed it was an error.

Reply:
It's ok, they embedded the css in the js. Shoot the symlink :+1: